src/reanimated2/hook/useScrollViewOffset.ts

```diff
@@ -0,0 +1,36 @@
+import { RefObject, useEffect, useRef } from 'react';
+
+import { findNodeHandle } from 'react-native';
+import type Animated from 'react-native-reanimated';
+import { useEvent, useSharedValue } from '.';
+import { SharedValue } from '../commonTypes';
+import { ScrollEvent } from './useAnimatedScrollHandler';
+
+const subscribeForEvents = [
```

Comment:
We should use verbs for functions or methods and nouns for variables etc. I would suggest `eventNames` instead of `subscribeForEvents`.